app/views/declaration/summary/summary_page.scala.html

```diff
@@ -32,20 +32,22 @@
     title = messages("supplementary.summary.title")
 ) {
 
-    @if(mode == Mode.Draft) {
-        @components.back_link(controllers.routes.SavedDeclarationsController.displayDeclarations(), mode)
-    } else {
-
-        @{
-            if(request.choice.value == StandardDec)
-                components.back_link(controllers.declaration.routes.SealController.displayPage(mode), mode)
-            else {
-                if(suppDecData.transportInformationContainerData.fold(false)(_.containers.nonEmpty))
-                    components.back_link(controllers.declaration.routes.TransportContainerController.displayPage(mode), mode)
-                else
-                    components.back_link(controllers.declaration.routes.TransportDetailsController.displayPage(mode), mode)
-            }
-        }
+  @mode match {
+      case Mode.Draft => {
+          @components.back_link(controllers.routes.SavedDeclarationsController.displayDeclarations(), mode)
+      }
+      case Mode.Amend => {
+          @components.back_link(controllers.routes.SavedDeclarationsController.displayDeclarations(), mode)
+      }
```

Comment:
Aren't those two cases doing the same thing?

Reply:
Good spot, think that was a copy and paste error